fileshare/fileshare.go

```diff
@@ -5,14 +5,10 @@ import (
 	"net/netip"
 	"time"
 
+	norddrop "github.com/NordSecurity/libdrop-go/v7"
 	"github.com/NordSecurity/nordvpn-linux/fileshare/pb"
 )
 
-const (
-	DirDepthLimit     = 5
-	TransferFileLimit = 1000
```

Comment:
These constants define the feature (fileshare), not the implementation (libdrop), therefore it is OK for them to stay in the `fileshare` package. Migration caused usage of `libdrop` package in multiple places, therefore a requirement for native dependency availability (`libdrop.so`).

It is OK for specific implementation (libdrop) to know about the feature it implements (fileshare) in this case in order to avoid introducing additional abstraction layers.

Reply:
Moved it back. I moved it here, because of circular dependency issue:
```
package github.com/NordSecurity/nordvpn-linux/cmd/fileshare
        imports github.com/NordSecurity/nordvpn-linux/fileshare
        imports github.com/NordSecurity/nordvpn-linux/fileshare/libdrop
        imports github.com/NordSecurity/nordvpn-linux/fileshare: import cycle not allowed
```
but I moved the `EventToString` helper from libdrop package to fileshare package, so the part 
```
imports github.com/NordSecurity/nordvpn-linux/fileshare
        imports github.com/NordSecurity/nordvpn-linux/fileshare/libdrop
```
is no longer valid.
This helper method is only used in `even_manager.go` so we can have it there I think